windows-nsis/openvpn.nsi

```diff
@@ -454,6 +454,26 @@ Function .onInit
 	${GetParameters} $R0
 	ClearErrors
 
+${IfNot} ${AtLeastWinVista}
+
+	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDNO guiEndNo
+	DetailPrint "Downloading the latest WinXP build ..."
+	Goto guiEndYes
+
+	guiEndNo:
+	Quit
+
+	guiEndYes:
+	${If} ${RunningX64}
+		ExecShell "open" "http://build.openvpn.net/downloads/releases/latest/openvpn-install-latest-winxp-x86_64.exe" 
+	${Else}
+		ExecShell "open" "http://build.openvpn.net/downloads/releases/latest/openvpn-install-latest-winxp-x86_64.exe"
```

Comment:
sorry for copy/paste

as installer itself is signed, I consider http as acceptable, @mattock ?